,gitignore

```diff
@@ -0,0 +1,6 @@
+# Folders
+node_modules/
+
+# Files
+package-lock.json
+package.json
```

Comment:
Please check the ".gitignore" file

Reply:
I think it would be easy to understand if you comment like: Please rename your `.gitignore` file. It should be `.gitignore` instead of `,gitignore`.